components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -514,5 +520,15 @@ public void setManagementApp(boolean managementApp) {
 
         isManagementApp = managementApp;
     }
+
+    public boolean isB2BSelfServiceApp() {
+
+        return isB2BSelfServiceApp;
+    }
+
+    public void setB2BSelfServiceApp(boolean b2BSelfServiceApp) {
```

Comment:
```suggestion
    public void setB2BSelfServiceApp(boolean b2bSelfServiceApp) {
```
Is `b2BSelfServiceApp` an auto-generated variable name? for me it's difficult to read. `b2bSelfServiceApp` would give  better readability. For abbreviation of 3 letters or less we can follow the same case right